tests/core/editable/domfix.js

```diff
@@ -8,6 +8,7 @@ bender.test( {
 	'async:init': function() {
 		var tc = this;
 		var editor = new CKEDITOR.editor();
+
```

Comment:
Why such changes were introduced in this PR?

Reply:
I've remove this chaneg together with simmilar cases in other files.